lls/main.c

```diff
@@ -500,12 +531,12 @@ lls_stage2(void *arg)
 	 * Gatekeeper or Grantor.
 	 */
 
-	/*
-	 * TODO Have a different block set up RSS on the back interface,
-	 * and classify + distribute ND packets to the LLS block using
-	 * the DPDK libraries.
-	 */
-	if (lls_conf->nd_cache.iface_enabled(net_conf, &net_conf->back)) {
+	ret = register_nd_acl_rules(&net_conf->front);
+	if (ret < 0)
+		return ret;
+
+	/* TODO Have a different block set up RSS on the back interface. */
```

Comment:
Ok, let's wait to find the need for it.

I was thinking that there could be a setup in which no block sets RSS at the back. And as you pointed, this could eventually happen to the front as well.